src/applications/disability-benefits/all-claims/tests/pages/vaMedicalRecords.unit.spec.jsx

```diff
@@ -26,13 +31,22 @@ describe('VA Medical Records', () => {
     },
   ];
 
-  it('should render', () => {
+  const newDisabilities = [
+    {
+      cause: 'NEW',
+      condition: 'asthma',
+      'view:descriptionInfo': {},
+    },
```

Comment:
Should this include `'view:selected': true`? I can't remember if you can unselect new issues.

Reply:
Yes, user can unselect a new condition on the VA Medical Records page. However, that boolean gets stored in the `vaTreatmentFacilities` array